stingray/modeling/posterior.py

```diff
@@ -315,16 +330,18 @@ def evaluate(self, pars, neg=False):
 
         mean_model = self.model(self.x)
 
-        if self.m == 1:
-            loglike = -np.sum(np.log(mean_model)) - \
-                      np.sum(self.y/mean_model)
+        with warnings.catch_warnings(record=True) as out:
```

Comment:
Pretty much. Occasionally during fitting or MCMC, the model ends up with a parameter set that returns values not allowed by the likelihood (like negative numbers in logs), which makes for very messy output.